pages/index.vue

```diff
@@ -2,16 +2,16 @@
 import _ from 'underscore'
 import type { User } from '~/libs/apiTypes'
 import { getAsyncDataOrThrows } from '~/libs/getAsyncData'
-import type { Projects } from '~/libs/types'
+import type { Project } from '~/libs/types'
```

Comment:
Can we group types under `/libs/types` ? Instead of having them spreaded betwen apiTypes (which is only about user by the way) and types ?

Reply:
ok